src/Robo/Commands/Vm/VmCommand.php

```diff
@@ -328,15 +328,23 @@ protected function customizeConfigFiles() {
   protected function createConfigFiles() {
     $this->logger->info("Creating configuration files for Drupal VM...");
 
-    $this->taskFilesystemStack()
-      ->mkdir($this->vmDir)
-      ->copy($this->defaultDrupalVmConfigFile, $this->projectDrupalVmConfigFile,
-        TRUE)
-      ->copy($this->defaultDrupalVmVagrantfile,
-        $this->projectDrupalVmVagrantfile, TRUE)
-      ->stopOnFail()
-      ->setVerbosityThreshold(VerbosityThresholdInterface::VERBOSITY_VERBOSE)
-      ->run();
+    $configFiles = [
+      $this->projectDrupalVmConfigFile => $this->defaultDrupalVmConfigFile,
+      $this->projectDrupalVmVagrantfile => $this->defaultDrupalVmVagrantfile,
+    ];
+    foreach ($configFiles as $projectConfigFile => $defaultConfigFile) {
+      // Skip projects' existing configuration files.
+      if (file_exists($projectConfigFile)) {
+          continue;
+      }
+
+      $this->taskFilesystemStack()
+          ->mkdir($this->vmDir)
```

Comment:
If the box/config.yml file doesn't exist, then we need to create the box directory (Probably. There's an odd chance that the box directory could exist but not the config.yml file, but [Symfony's mkdir operation](https://symfony.com/doc/current/components/filesystem.html#mkdir) will just ignore it if we tell it to create a dir that already exists.). But if the Vagrantfile doesn't exist, we don't need to create the box dir.

We'll also get a bit better performance if we just execute the taskFilesystemStack()->run() command once, so could you pull that out of the foreach loop?